internal/controller/standalone_pgadmin/users.go

```diff
@@ -231,16 +251,24 @@ cd $PGADMIN_DIR
 					intentUsers = append(intentUsers, existingUser)
 					continue
 				}
+				// If update user fails due to user not found or password length:
+				// https://github.com/pgadmin-org/pgadmin4/blob/REL-8_5/web/setup.py#L263
+				// https://github.com/pgadmin-org/pgadmin4/blob/REL-8_5/web/setup.py#L246
+				if strings.Contains(stdout.String(), "User not found") ||
+					strings.Contains(stdout.String(), "Password must be") {
+
+					log.Info(fmt.Sprintf("Failed to update pgAdmin user %s: %s",
```

Comment:
:pick: 
`fmt.Sprintf("Failed to update pgAdmin user %s: %s", intentUser.Username, stdout.String())` could be pulled out and stored in a variable so any future changes only need to be made once.